### PR TITLE
python-utils recipe

### PR DESCRIPTION
**Pull request by maartenbreddels**
@WoLpH you only specify BSD for a license, is that BSD-3-Clause?

### CODE REVIEW

**Comment by WoLpH** [comment]
@maartenbreddels yes, that's indeed the 3-clause bsd license

File: recipes/python-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - python_utils
+    - python_utils.converters
+    - python_utils.formatters
+
+about:
+  home: https://github.com/WoLpH/python-utils
+  license: BSD
```
Comment (maartenbreddels) [inline recipes/python-utils/meta.yaml]: 👍

**Comment by WoLpH** [comment]
Awesome, thanks for the quick merge @patricksnape and thanks to @maartenbreddels for doing all the work :)

File: recipes/python-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  summary: 'Python Utils is a collection of small Python functions and classes which make common patterns shorter and easier.'
+  description: |
+    Python Utils is a collection of small Python functions and classes which make common patterns shorter and easier. It is by no means a complete collection but it has served me quite a bit in the past and I will keep extending it.
+    One of the libraries using Python Utils is Django Utils.
+  doc_url: http://python-utils.readthedocs.io/
+  dev_url: https://github.com/WoLpH/python-utils
+
+extra:
+  recipe-maintainers:
+    - MaartenBreddels
```
Comment (jakirkham) [inline recipes/python-utils/meta.yaml]: Should have been lower cased. Please fix in the feedstock.

**Pull request merged**